src/core/linux/SDL_dbus.c

```diff
@@ -183,14 +183,12 @@ void SDL_DBus_Quit(void)
         dbus.connection_close(dbus.session_conn);
         dbus.connection_unref(dbus.session_conn);
     }
-/* Don't do this - bug 3950
```

Comment:
Please don't remove this comment. Can you add the information from @smcv about unexpected uses of d-bus in your application?

Reply:
Actually, can you move this comment and @smcv's info to the hint in the header?

Reply:
Sorry for the delay - I tried to summarize it in the header already, but I'm not sure about the wording. Which info would you like me to add, and how should I word it?